### ISSUE TITLE
Resolution of Video settings

### ISSUE THREAD

**Issue by PickleNik** [PickleNik/VueTube#16]
From VueTube created by [ToTheMoon1234](https://github.com/ToTheMoon1234): VueTubeApp/VueTube#79  

![Screenshot_20220317-194229~2](https://user-images.githubusercontent.com/73231423/158826996-ee92bcd0-e2a0-41de-bfe4-2092ee47c6ea.png)

An option to :
-the old quality selecter ( ie resolution selector like 144p 240p.....1080p instead of options like high quality etc, like in the new one) 

-Forced/Preferred resolution over mobile data for all videos

A lot of people(like myself) have data caps so I think an option to always force set all the videos to a specific resolution over mobile data is a must for the app. 

-Forced/Preferred resolution over wifi for all videos

The default YouTube data selector(both old and new) is  for current video and is not reliable as it sometimes sets the resolution of the video above or below the specified setting. Also enabling the data saver settings only sets resolution of all videos to 144p and user can't choose what he wants say 240p or 720p for all videos. 

So a setting like preferred resolution over mobile data/wifi is needed from the app side like in vanced 

Basically the given options in the attached image from vanced

Edit: 'Forcing to set' the the resolution will be more accurate description than 'preferred'
